menpowidgets/base.py

```diff
@@ -766,17 +767,14 @@ def render_function(change):
             figure_size=new_figure_size, **options)
 
         # Update info
-        if custom_function is None:
-            update_info(images[im], image_is_masked, selected_group)
-        else:
-            update_info(images[im], image_is_masked, selected_group,
-                        custom_function(images[im]))
+        update_info(images[im], image_is_masked, selected_group,
+                    custom_info_callback)
 
         # Save the current figure id
         save_figure_wid.renderer = renderer
 
     # Define function that updates the info text
-    def update_info(img, image_is_masked, group, custom_str=None):
+    def update_info(img, image_is_masked, group, custom_info_callback):
```

Comment:
Tiny nitpick. If something is expected to *maybe* be `None`, we generally use keyword arguments. This lets the developer know that the argument is optional. So, although its a touch verbose, it would be good if this was `custom_info_callback=custom_info_callback`